releasenotes/notes/remove-transpile-broadcast-1dfde28d508efa0d.yaml

```diff
@@ -0,0 +1,43 @@
+---
+upgrade:
+  - |
+    Support for passing in lists of argument values to the :func:`~.transpile`
+    function is removed. This functionality was deprecated as part of the
+    0.23.0 release and is now being removed. Removing this functionality was
+    necessary to greatly reduce the overhead for parallel execution for
+    transpiling multiple circuits at once. If you’re using this functionality
+    currently you can call :func:`~.transpile` multiple times instead. For
+    example if you were previously doing something like::
```

Comment:
We maybe want to make a little clearer that passing in a list of circuit inputs is still permitted (for the parallelisation).

Reply:
Updated in: https://github.com/Qiskit/qiskit-terra/pull/10291/commits/ffd10191defea7300c659fa1c4e3aa3187be3a26